connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -58,18 +61,21 @@
 
     private Map<String, String> config;
     private State state;
-
-    public WorkerConnector(String connName,
-                           Connector connector,
-                           ConnectorContext ctx,
-                           ConnectMetrics metrics,
-                           ConnectorStatus.Listener statusListener) {
+    private final OffsetStorageReader offsetStorageReader;
+
+    public WorkerConnector(final String connName,
```

Comment:
adding `final` in every argument here was just noise but with not much added value. Given that we don't write constructors like this in Connect I think we should revert and add the new argument without the `final` specifier.